apmpackage/cmd/gen-package/paths.go

```diff
@@ -20,23 +20,23 @@ package main
 import "path/filepath"
 
 func docsTemplateFilePath(version string) string {
-	return filepath.Join("apmpackage/apm/", version, "/_dev/docs/README.template.md")
```

Comment:
Hmmm... I thought that `_dev` was a convention for things that don't actually belong to the published package. 
Probably too far fetched, but I worry a bit about this change. Imagine eg. that instead of using `README.md` Kibana starts reading all `*.md` files and show only the first one, or concats them, etc.